.gitignore

```diff
@@ -327,7 +325,6 @@ celerybeat.pid
 
 # Environments
 .env
-.venv
```

Comment:
Why did you remove the VirtualEnv folder?

Reply:
I removed those items because they are duplicated. For example, .venv is already defined on line 139.

Reply:
I see. 👍🏻